### PR TITLE
chore: Remove archived or deleted repos

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: repos-github.md [hunk 1]
```diff
@@ -1438,7 +1433,6 @@
 - wiringbits/cazadescuentos
 - wiringbits/my-photo-timeline
 - wiringbits/safer.chat
-- wiringbits/scala-webapp-template
```
Comment (mkurz) [inline repos-github.md:1441]: @AlexITC Not sure if you are aware that they removed scala-steward from your repo here. Also see https://github.com/VirtusLab/scala-steward-repos/issues/98

Comment (mkurz) [reply repos-github.md:1441]: Probably it failed at some point so they removed it? Maybe re-add it again...

Comment (AlexITC) [reply repos-github.md:1441]: Thanks for notifying me, I guess they gave up on https://github.com/wiringbits/scala-webapp-template/issues/298

Comment (tgodzik) [reply repos-github.md:1441]: Which might explain `Weirdly enough I can't find anything in the logs now :/ I will try to run it again.` :facepalm: 

Sorry about that! I had a grand ambition of fixing all the repos, but in the end I never had time to finish this.

Comment (AlexITC) [reply repos-github.md:1441]: By running my own steward instance, the PRs started flowing again https://github.com/wiringbits/scala-webapp-template/pull/329

**Pull request merged**